remoteappmanager/application.py

```diff
@@ -181,11 +191,45 @@ def _reverse_proxy_init(self):
             self._config.proxy_api_url
         ))
 
+    def _db_init(self):
+        """Initializes the database connection."""
+        self.db = orm.Database(self.config.db_url)
+
+    def _user_init(self):
+        Session = self.db.create_session_factory()
+
+        session = Session()
+        try:
+            user = session.query(orm.User).filter_by(
+                name=self.config.user).one_or_none()
+        except MultipleResultsFound:
```

Comment:
Unsure if this is needed. The point is that if you get multiple users out of this query, something really wrong happened at the schema definition, or we are using a database that doesn't enforce unique keys.